src/test/kotlin/qbit/model/EntityTest.kt

```diff
@@ -82,6 +82,14 @@ class EntityTest {
         assertEquals("s2", e[ref][s])
     }
 
+    @Test
+    fun testEmptyRefList() {
+        val ref = RefListAttr(root["ref"], false)
+        val e = AttachedEntity(EID(0), mapOf(ref to ArrayList<EID>()), emptyDb, false)
```

Comment:
Such behavior (empty reflist) is impossible with current storage format
It's better to add check, that list is not empty in AttachedEntity.tryGet and expect AssertionError in this test

Reply:
Temporarily removed this test